torch/csrc/jit/init.cpp

```diff
@@ -117,14 +118,44 @@ void initJITBindings(PyObject* module) {
           [](std::shared_ptr<Graph>& g) { return PropagateQuantInfo(g); })
       .def(
           "_jit_pass_insert_observers",
-          [](std::shared_ptr<Graph>& g, py::function pyObserverFunction) {
+          [](std::shared_ptr<script::Module>& moduleObj,
```

Comment:
That should probably be inside `InsertObserverNodes` function. Also, it probably should be based on #19386.

Reply:
All meaningful code should ideally be out of `init.cpp` - this file is for bindings.